filament/backend/src/metal/MetalEnums.h

```diff
@@ -71,7 +71,7 @@ constexpr inline MTLIndexType getIndexType(size_t elementSize) noexcept {
     } else if (elementSize == 4) {
         return MTLIndexTypeUInt32;
     }
-    FILAMENT_CHECK_POSTCONDITION(false) << "Index element size not supported.";
+    FILAMENT_CHECK_POSTCONDITION(false) << "Index element size " << elementSize << " not supported.";
```

Comment:
we really shouldn't have a `FILAMENT_CHECK` in a method like this. This type of bugs should be handled on the filament side, and just for safety, we can have an assert_invariant() here (DEBUG only, really just for to catch really serious bugs).

Reply:
Agreed. I'll switch it to a `assert_invariant()`.